src/components/TaskList.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import '../css/index.css'
+import cross from '../assets/icons/close.png';
+import { useReminders, List } from '../utils/RemindersContext';
+
+const TaskList = ({ id, name }: List) => {
+  const {
+    lists, setLists, setActiveListId
+  } = useReminders();
+
+  if (id.length < 10) {
```

Comment:
Once again using the id length to control the logic, but this time using `10` instead of `12` (which demonstrates the issue)